Resources/public/js/views/services/plugins/ez-objectrelationsloadplugin.js

```diff
@@ -81,7 +143,17 @@ YUI.add('ez-objectrelationsloadplugin', function (Y) {
              */
             contentModelConstructor: {
                 value: Y.eZ.Content
-            }
+            },
+
+            /**
+             * Location constructor
+             *
+             * @attribute locationModelConstructor
+             * @type Y.eZ.Location
+             */
+            locationModelConstructor: {
+                value: Y.eZ.Location
```

Comment:
this should be reflected in `yui.yml`, the Location model is now a dependency of this plugin.